WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -2017,9 +2017,14 @@ - (BOOL)assetsPickerController:(CTAssetsPickerController *)picker shouldSelectAs
                                 message:NSLocalizedString(@"This image is currently not available locally to be used in your post. Please try again later.", @"User information explaining that the image is not available locally. This is normally related to share photo stream images.")];
```

Comment:
@meremagee another copy check (same as above). Please ignore me if this has already been hashed over! :grinning: